File: web3rpc/rpc-specs/code-samples/python/debug/logging/vmodule.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

module = "p2p=4"

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.vmodule(module)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/vmodule.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const module = "p2p=4";

  provider.debug
    .vmodule(module, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
